swift/StableDiffusion/pipeline/StableDiffusionPipeline.swift

```diff
@@ -56,20 +53,19 @@ public struct StableDiffusionPipeline: ResourceManaging {
     ///   - unet: Model for noise prediction on latent samples
     ///   - decoder: Model for decoding latent sample to image
     ///   - safetyChecker: Optional model for checking safety of generated images
-    ///   - guidanceScale: Influence of the text prompt on generation process
     ///   - reduceMemory: Option to enable reduced memory mode
     /// - Returns: Pipeline ready for image generation
-    public init(textEncoder: TextEncoder,
-                unet: Unet,
-                decoder: Decoder,
-                safetyChecker: SafetyChecker? = nil,
-                guidanceScale: Float = 7.5,
-                reduceMemory: Bool = false) {
+    public init(
+        textEncoder: TextEncoder,
```

Comment:
Could you please maintain the previous indentation and only remove the relevant argument?

Reply:
Done :D